src/main/kotlin/apply/domain/mission/Mission.kt

```diff
@@ -42,6 +49,9 @@ class Mission(
     @Column(nullable = false)
     private var deleted: Boolean = false
 
+    val submittable: Boolean
+        get() = now().isBetween(submitPeriod)
```

Comment:
```suggestion
        get() = submitPeriod.contains(now())
        // 이름이 애매하다면..? submitPeriod.isWithin(now())
```

c: `MissionSubmitPeriod` 내에 확장함수 없애고 위와 같이 작성하는 것이 더 자연스러워 보이는데 어떻게 생각하시나요?